src/librustc_codegen_ssa/back/write.rs

```diff
@@ -1679,7 +1679,6 @@ impl SharedEmitterMain {
                         d.code(code);
                     }
                     handler.emit_diagnostic(&d);
-                    handler.abort_if_errors_and_should_abort();
```

Comment:
Just to double check: we still do make sure that errors occur before codegen that we won't emit an output program?

Reply:
This is probably done by some unconditional `abort_if_errors` in `rustc_driver` or some other high-level code.
I'm pretty sure this particular line didn't do anything because the "`and_should_abort`" part was always `false` outside of the parser.